posthog/utils.py

```diff
@@ -354,7 +354,10 @@ def render_template(template_name: str, request: HttpRequest, context: Dict = {}
     context["posthog_app_context"] = json.dumps(posthog_app_context, default=json_uuid_convert)
 
     if posthog_distinct_id:
-        feature_flags = posthoganalytics.get_all_flags(posthog_distinct_id, only_evaluate_locally=True)
+        groups = {}
+        if request.user and request.user.is_authenticated:
+            groups = {"organization": request.user.organization_id}
```

Comment:
Avoiding any `Int` vs `string` comparison schenanigans, as local evaluation is sensitive to types

```suggestion
            groups = {"organization": str(request.user.organization_id)}
```